test/integration_test/basic_test.go

```diff
@@ -10,19 +10,18 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/require"
 	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	appsops "github.com/portworx/sched-ops/k8s/apps"
+	coreops "github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/sched-ops/k8s/operator"
```

Comment:
nit: format the imports

Reply:
isnt this already formatted? this group is for portworx

Reply:
The practice is not to format the imports for every group. Generally there are 2 or 3 groups - 
- in-built packages
- external package
- packages internal to this repository. In our case - `github.com/libopenstorage/operator/...`

Reply:
this section was actually auto-generated by intellij, I think it detected the code and figured out the import and alias.

Reply:
actually there were there before my PR, so somehow intellij decided to move them here. let's compare our intellij settings in todays standup meeting.

Reply:
If you remove the blank lines between those imports, intellij should automatically group them and sort it.

Reply:
sounds good, done